snap/snapcraft.yaml

```diff
@@ -212,12 +232,12 @@ parts:
       cd ${CRAFT_PART_SRC}/3rd-party
       find . -name 'LICENSE*' -print0 | xargs -0 -I FILE install --mode=644 -D --no-target-directory FILE ${CRAFT_PART_INSTALL}/licenses/FILE
 
-      # Stage protoc and protobuf files for use in the gui part
-      PROTOBUF_INSTALL_DIR=${CRAFT_PART_INSTALL}/protobuf
-      mkdir -p ${PROTOBUF_INSTALL_DIR}/google/protobuf
-      cp -a ${CRAFT_PART_BUILD}/vcpkg_installed/*/tools/protobuf/protoc* ${PROTOBUF_INSTALL_DIR}
-      cp -a ${CRAFT_PART_BUILD}/vcpkg_installed/*/include/google ${PROTOBUF_INSTALL_DIR}
-      cp -a ${CRAFT_PART_SRC}/src/rpc/multipass.proto ${PROTOBUF_INSTALL_DIR}
+      # # Stage protoc and protobuf files for use in the gui part
```

Comment:
Why does this part remain commented? Should we just remove them?

Reply:
True. I commented them to test something but then I forgot to remove them. My bad.